### ISSUE TITLE
Check that resulting image is tinier

### ISSUE THREAD

**Issue by sianhulo** [kornelski/pngquant#311]
I don't know if this will be considered a valid complaint, as pngquant doesn't overwrite the converted file by default. I use the following command `pngquant -f --ext .png` as the quality loss isn't important for my usage, however, it has come to my attention that some files will actually end up being bigger than the original file, seems to mostly (if not exclusively) happen with gray scale images in my experience. So I don't know if you'd consider adding a check to determine if the resulting image is bigger than the source image before actually overwriting it.

**Comment by kornelski** [comment]
There is `--skip-if-larger` option. 

I suppose it would be nice if it was the default, but then such options are usually disabled with `--force` :/

**Comment by sianhulo** [comment]
Yeah, I forgot the setting existed, though I have actually have it in my alias, simply because, as you said, it is disabled by --force.

**Comment by kornelski** [comment]
Hmm, so adding `--overwrite` might be the solution
